### PR TITLE
Create weekly-update.2015-05-15.md

### PR DESCRIPTION
**Pull request by Gioyik**
CC #71

### CODE REVIEW

**Comment by yosuke-furukawa** [comment]
This weekly update is the most important updates from io.js historical perspective.
I would like to show [the news](https://twitter.com/rvagg/status/598605393636429825) more bigger.

like this:

# io.js News

We are happy to speak this news, **"io.js decides to join the Node Foundation"**.
And our organization name will be renamed `nodejs`.

As a first step, we will move from `iojs` organization to `nodejs` organization and will converge `joyent/node` gradually. 
We will continue to release io.js until the convergence have done.

If you would like to see the convergence plan in detail, please confirm [this link](https://github.com/jasnell/dev-policy/blob/master/convergence.md).
When you would like to ask questions, feel free to ask [@Fishrock123](fishrock123@rocketmail.com). 

The discussion log is [here](https://github.com/nodejs/io.js/issues/1664).
Please wait to release `new node` a little more :)

# io.js release

This week we hadn't io.js release.

### Community Updates
- [Performance Showdown](https://raygun.io/blog/2015/05/performance-showdown-node-js-vs-io-js-v2-0-0/): Node.js vs. io.js v2.0.0.
- Microsoft will propose Node run different JavaScript backend including `Chakra`.
- {node,io}.js [benchmark graphs](http://thlorenz.com/benchgraph/).
- [TC members](https://twitter.com/rvagg/status/598605393636429825) decide to move io.js to Node Foundation and the organization name will be renamed nodejs.

### Upcoming Events
- [NodeConf Adventure](http://nodeconf.com/) tickets are on sale, June 11th - 14th at Walker Creek Ranch, CA
- [CascadiaJS](http://2015.cascadiajs.com/) tickets are on sale, July 8th - 10th at Washington State
- [BrazilJS Conf](http://braziljs.com.br/) tickets are on sale, August 21st - 22nd at Shopping Center BarraShoppingSul
- [NodeConf EU](http://nodeconf.eu/) tickets are on sale, September 6th - 9th at Waterford, Ireland

**Comment by Gioyik** [comment]
It's done @yosuke-furukawa. Thank you a lot, let me know what you think

File: weekly-updates/weekly-update.2015-05-15.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# io.js News
+
+We are happy to speak this news, **"io.js decides to join the Node Foundation"**. And our organization name will be renamed `nodejs`.
+
+As a first step, we will move from `iojs` organization to `nodejs` organization and will converge `joyent/node` gradually. We will continue to release `io.js` until the convergence have done.
+
+If you would like to see the convergence plan in detail, please check it at [jashkell repo](https://github.com/jasnell/dev-policy/blob/master/convergence.md). When you would like to ask questions, feel free to ask [@Fishrock123](https://github.com/nodejs/evangelism/pull/fishrock123@rocketmail.com). The discussion log is [here](https://github.com/nodejs/io.js/issues/1664).
```
Comment (yosuke-furukawa) [inline weekly-updates/weekly-update.2015-05-15.md]: oops, fishrock email is not correct ...

**Comment by Gioyik** [comment]
fix problem

**Comment by mikeal** [comment]
the org has been renamed.

**Comment by Gioyik** [comment]
Done!

File: weekly-updates/weekly-update.2015-05-15.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# io.js News
+
+We are happy to speak this news, **"io.js decides to join the Node Foundation"**. And our organization name will be renamed `nodejs`.
+
```
Comment (yosuke-furukawa) [inline weekly-updates/weekly-update.2015-05-15.md:4]: And our organization name has been renamed `nodejs`.

File: weekly-updates/weekly-update.2015-05-15.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+As a first step, we will move from `iojs` organization to `nodejs` organization and will converge `joyent/node` gradually. We will continue to release `io.js` until the convergence have done.
```
Comment (yosuke-furukawa) [inline weekly-updates/weekly-update.2015-05-15.md:5]: As a first step, we have moved from `iojs` organization to `nodejs` organization

**Comment by Gioyik** [comment]
you're right @yosuke-furukawa. I forget it.

**Comment by yosuke-furukawa** [comment]
LGTM

File: weekly-updates/weekly-update.2015-05-15.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+Please wait for new node release a little more :)
+
+# io.js release
+This week we hadn't io.js release.
+
+### Community Updates
+
+* [Performance Showdown](https://raygun.io/blog/2015/05/performance-showdown-node-js-vs-io-js-v2-0-0/): Node.js vs. io.js v2.0.0.
+* Microsoft will propose Node run different JavaScript backend including `Chakra`.
```
Comment (mikeal) [inline weekly-updates/weekly-update.2015-05-15.md]: Microsoft has node.js running on their JavaScript VM `Chakra`.

And we should probably link to the repo.

File: weekly-updates/weekly-update.2015-05-15.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# io.js News
+
+We are happy to speak this news, **"io.js decides to join the Node Foundation"**. And our organization name will be renamed `nodejs`.
+
+As a first step, we will move from `iojs` organization to `nodejs` organization and will converge `joyent/node` gradually. We will continue to release `io.js` until the convergence have done.
+
+If you would like to see the convergence plan in detail, please check it at [jashkell repo](https://github.com/jasnell/dev-policy/blob/master/convergence.md). When you would like to ask questions, feel free to ask [@Fishrock123](mailto:fishrock123@rocketmail.com). The discussion log is [here](https://github.com/nodejs/io.js/issues/1664).
```
Comment (mikeal) [inline weekly-updates/weekly-update.2015-05-15.md]: this is being renamed to `nodejs/node` as we speak. might want to hold off until it is.

File: weekly-updates/weekly-update.2015-05-15.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+We are happy to speak this news, **"io.js decides to join the Node Foundation"**. And our organization name will be renamed `nodejs`.
+
+As a first step, we will move from `iojs` organization to `nodejs` organization and will converge `joyent/node` gradually. We will continue to release `io.js` until the convergence have done.
+
+If you would like to see the convergence plan in detail, please check it at [jashkell repo](https://github.com/jasnell/dev-policy/blob/master/convergence.md). When you would like to ask questions, feel free to ask [@Fishrock123](mailto:fishrock123@rocketmail.com). The discussion log is [here](https://github.com/nodejs/io.js/issues/1664). 
+
+Please wait for new node release a little more :)
+
+# io.js release
+This week we hadn't io.js release.
```
Comment (mikeal) [inline weekly-updates/weekly-update.2015-05-15.md]: Let's just not have the io.js release section this week :)

**Comment by mikeal** [comment]
I know it's a little late but I would like to get links in to all the working group and TC meetings this week.

[Hardware WG](https://www.youtube.com/watch?v=EFJVMGRtDWE)
[io.js/node.js Build Convergence](https://www.youtube.com/watch?v=8dxkM9vHmrY)
[TC Meeting](https://www.youtube.com/watch?v=UbYiFLf7MpU)

**Comment by Gioyik** [comment]
Please review it again. Changes are done!

File: weekly-updates/weekly-update.2015-05-15.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
+# io.js News
+
+We are happy to speak this news, **"io.js decides to join the Node Foundation"**. And our organization name will be renamed `nodejs`.
+
+As a first step, we will move from `iojs` organization to `nodejs` organization and will converge `joyent/node` gradually. We will continue to release `io.js` until the convergence have done.
+
+If you would like to see the convergence plan in detail, please check it at [jashkell repo](https://github.com/jasnell/dev-policy/blob/master/convergence.md). When you would like to ask questions, feel free to ask [@Fishrock123](mailto:fishrock123@rocketmail.com). The discussion log is [here](https://github.com/nodejs/node/issues/1664).
```
Comment (julianduque) [inline weekly-updates/weekly-update.2015-05-15.md]: s/jashkell/jasnell

File: weekly-updates/weekly-update.2015-05-15.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
+# io.js News
+
+We are happy to speak this news, **"io.js decides to join the Node Foundation"**. And our organization name will be renamed `nodejs`.
+
+As a first step, we will move from `iojs` organization to `nodejs` organization and will converge `joyent/node` gradually. We will continue to release `io.js` until the convergence have done.
+
+If you would like to see the convergence plan in detail, please check it at [jashkell repo](https://github.com/jasnell/dev-policy/blob/master/convergence.md). When you would like to ask questions, feel free to ask [@Fishrock123](mailto:fishrock123@rocketmail.com). The discussion log is [here](https://github.com/nodejs/node/issues/1664). 
+
+Please wait for new node release a little more :)
```
Comment (julianduque) [inline weekly-updates/weekly-update.2015-05-15.md]: I think this parragraph is unnecessary here ;)

**Comment by Gioyik** [comment]
Changes made

File: weekly-updates/weekly-update.2015-05-15.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
+# io.js News
+
+We are happy to speak this news, **"io.js decides to join the Node Foundation"**. And our organization name will be renamed `nodejs`.
```
Comment (diagramatics) [inline weekly-updates/weekly-update.2015-05-15.md]: > happy to speak this news

"announce" instead of speak, or something like "happy to announce this"?

**Comment by yosuke-furukawa** [comment]
Merged!

**Pull request merged**